src/preseed/config.cfg

```diff
@@ -2,8 +2,8 @@ d-i base-installer/kernel/linux/initramfs-generators string initramfs-tools
 
 # We don't want use a remote mirror (all stuff we need is on the USB flash drive)
 d-i base-installer/install-recommends boolean true
-d-i apt-setup/non-free boolean false
-d-i apt-setup/contrib boolean false
+d-i apt-setup/non-free boolean true
```

Comment:
non-free.. ewww... do we really need it? why?

Reply:
intel-microcode